tools/autograd/derivatives.yaml

```diff
@@ -558,7 +558,7 @@
 - name: index.Tensor(Tensor self, Tensor?[] indices) -> Tensor
   self: index_backward(grad.new_zeros(self.sizes(), self.options()), indices, grad)
 
-- name: index_add_(Tensor(a!) self, int dim, Tensor index, Tensor source) -> Tensor(a!)
+- name: index_add_(Tensor(a!) self, int dim, Tensor index, Tensor source, Scalar alpha=1) -> Tensor(a!)
   self: grad
   source: grad.index_select(dim, index).expand_as(source)
```

Comment:
maybe_multiply(..., alpha)

Reply:
Yes that's why I was asking about the role.
This formula definitely needs to be updated as natalia suggested.
I think we also want the test and doc to be updated.

Reply:
fixed